tensorflow/python/training/checkpoint_management.py

```diff
@@ -494,7 +494,7 @@ class CheckpointManager(object):
   """
 
   def __init__(self, checkpoint, directory,
-               max_to_keep, keep_checkpoint_every_n_hours=None):
+               max_to_keep, keep_checkpoint_every_n_hours=None, checkpoint_name="ckpt"):
```

Comment:
Nit: could you wrap the line to 80 characters?

Reply:
Gave each parameter a line of its own. Looks better and seems to be more aligned with the conventions.